sympy/polys/tests/test_constructor.py

```diff
@@ -121,3 +121,18 @@ def test_precision():
     result = construct_domain([f2])
     y = result[1][0]
     assert y-1 > 1e-50
+
+def test_issue_11538():
+    dom = ZZ[E]
+    assert construct_domain(E) == (dom, dom.convert(E))
```

Comment:
We may omit the `dom.convert` parts that are not of interest here. These two lines could be combined as follows:
`    assert construct_domain(E)[0] == ZZ[E]`

This also applies to later tests.